src/System.IO.Pipes/tests/AnonymousPipesSimpleTest.cs

```diff
@@ -36,18 +87,25 @@ public static void ClientSendsByteServerReceives()
         using (AnonymousPipeServerStream server = new AnonymousPipeServerStream(PipeDirection.In))
         {
             Assert.True(server.IsConnected);
+            server.ReadMode = PipeTransmissionMode.Byte;    // cover that path
```

Comment:
I'm not quite understanding this "cover" comments.  Were these lines just thrown in to boost code coverage?  if no such tests exist, it'd be better to have tests that not only execute this code but verify that the right things happened, e.g. that after setting ReadMode getting it returns the same value, that it has the appropriate impact on the rest of the library (if that's testable), that setting it to an invalid value yields the appropriate behavior, etc.  You might already have such tests, I was just thrown by the comment.